### PR TITLE
Update DemoApp for December SDK release

### PR DESCRIPTION
**Pull request by UdayaNalla**
## Description

Submitting code changes after "Setup DemoApp for testing with private CocoaPod/Bintray/NPM"

### CODE REVIEW

File: AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec [hunk 1]
```diff
@@ -8,7 +8,7 @@ Pod::Spec.new do |s|
 
   s.author           = { 'Microsoft' => 'appcentersdk@microsoft.com' }
 
-  s.source = { :http => "https://github.com/Microsoft/AppCenter-SDK-React-Native/releases/download/#{s.version}/AppCenter-SDK-ReactNative-iOS-Pod-#{s.version}.zip" }
```
Comment (bmourat) [inline AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec]: Please revert this change, this was only for publishing to private spec repo.

Comment (UdayaNalla) [reply AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec]: Requested changes done.
